### ISSUE TITLE
Build fails for cordova-ios 6.1.0 with this plugin

### ISSUE THREAD

**Issue by akshukla07** [apache/cordova-plugin-file-transfer#261]
# Bug Report
Unable to create build after updating to cordova-ios 6.1.0

## Problem
After updating to cordova-ios 6.1.0, I added the ios platform and tried created build with `ionic cordova build ios` command. This gives the following error:

`warning: Could not read serialized diagnostics file: Cannot Load File: Failed to open diagnostics file (in target 'APPNAME' from project 'APPNAME')

** ARCHIVE FAILED **

The following build commands failed:
	CompileC /Users/akhilesh/Library/Developer/Xcode/DerivedData/APPNAME-bjbhutjdzzhywsgeqawthgfleslm/Build/Intermediates.noindex/ArchiveIntermediates/APPNAME/IntermediateBuildFilesPath/APPNAME.build/Debug-iphoneos/APPNAME.build/Objects-normal/arm64/CDVFileTransfer.o /Users/akhilesh/github/app-folder/platforms/ios/APPNAME/Plugins/cordova-plugin-file-transfer/CDVFileTransfer.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
xcodebuild: Command failed with exit code 65
[ERROR] An error occurred while running subprocess cordova.
        
        cordova build ios exited with exit code 65`

### What is expected to happen?
The build process should complete without errors. When I remove this plugin, the app builds successfully.

### What does actually happen?
The build process throws error and exits.

`warning: Could not read serialized diagnostics file: Cannot Load File: Failed to open diagnostics file (in target 'APPNAME' from project 'APPNAME')

** ARCHIVE FAILED **

The following build commands failed:
	CompileC /Users/akhilesh/Library/Developer/Xcode/DerivedData/APPNAME-bjbhutjdzzhywsgeqawthgfleslm/Build/Intermediates.noindex/ArchiveIntermediates/APPNAME/IntermediateBuildFilesPath/APPNAME.build/Debug-iphoneos/APPNAME.build/Objects-normal/arm64/CDVFileTransfer.o /Users/akhilesh/github/app-folder/platforms/ios/APPNAME/Plugins/cordova-plugin-file-transfer/CDVFileTransfer.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
xcodebuild: Command failed with exit code 65
[ERROR] An error occurred while running subprocess cordova.
        
        cordova build ios exited with exit code 65`

## Information
The build process works fine if I remove the plugin. The issue only arises when this plugin is added.

### Command or Code
- Create an ionic 3 project.
- Add cordova-plugin-file-transfer: `ionic cordova plugin add cordova-plugin-file-transfer`
- Add cordova-ios@6.1.0: `ionic cordova platform add ios@6.1.0`
- Create build: `ionic cordova build ios`

### Environment, Platform, Device
Environment: Mac OS Catalina v10.15.5 (19F101) on MacBook Pro (Retina, 15-inch, Early 2013)
Platform: iOS 11.0 and above
Device: Creating build for iPhone 

### Version information
Ionic:

   Ionic CLI          : 5.4.16
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.4

Cordova:

   Cordova CLI       : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms : android 8.1.0, ios 6.1.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 25 other plugins)

Utility:

   cordova-res (update available: 0.15.1) : 0.10.0
   native-run (update available: 1.0.0)   : 0.2.9

System:

   Android SDK Tools : 26.1.1 (/Users/akhilesh/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.2
   NodeJS            : v10.16.3 (/usr/local/Cellar/node@10/10.16.3/bin/node)
   npm               : 6.9.0
   OS                : macOS Catalina
   Xcode             : Xcode 11.5 Build version 11E608c

## Checklist

- [x] I searched for existing GitHub issues
- [ ] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

**Comment by timbru31** [comment]
Duplicate of #258. Please note that this plugin is deprecated since 2017 and it's unlikely to receive a new update.

**Comment by sushibear** [comment]
The plugin is deprecated.. 

check this blog post:
[https://cordova.apache.org/blog/2017/10/18/from-filetransfer-to-xhr2.html](https://cordova.apache.org/blog/2017/10/18/from-filetransfer-to-xhr2.html
)

**Comment by breautek** [comment]
> The plugin is deprecated..
> 
> check this blog post: https://cordova.apache.org/blog/2017/10/18/from-filetransfer-to-xhr2.html

The file transfer actually has became un-deprecated but we haven't build a new release for it **yet**, so the current release is still the deprecated release. This plugin release is being blocked by some dependency, particularly the `cordova-plugin-file` plugin, which has failing tests.

The master branch of this plugin does have many issues resolved regarding usage on cordova-ios@6.

**Comment by Fourie-r** [comment]
Is this going to be released, i am still having troubles building for ios@6.2.0 ?

**Comment by pixellet14** [comment]
year 2022.....so what's the solution now?
